### PR TITLE
Feature/link doc

### PR DESCRIPTION
**Pull request by Crejak**
## Added an extension to consult doc for visualizing / creating links from associated documents :

- New "links" button on the document browser. It opens a new window which shows the links associated to the current document.
- For each link, two buttons are present, "highlight" (color the building) and "travel" (move the camera so that the building is centered in the screen)
  - To do the traveling, a `centroid` attribute was added to the building info in the TBI. It is calculated only once, when the tile is loaded for the first time.
  - The `focusCameraOn` method from `Geocoding` was moved to a `CameraUtils` file. It is used to move the camera so that it focuses on a specific point (exactly what we want to use here).
- Possibility to create a link in association to a document (button "select building" -> "create link")
- Deleted old `LinkVisualization` module, now useless

## Other changes / improvements

- Fixed a wrong import which made `npm run build` fail (thus made the project impossible to deploy)
- Made document & guided tour fetch requests asynchronous (no longer freeze the application while loading)
- Fixed the guided tour window when no guided tour was present in the database
- Added opacity slider for the preview image during document placement (#95)

### CODE REVIEW

File: UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+
+    // Adds the window to the view and hide it
+    this.appendTo(this.documentController.parentElement);
+    this.disable();
+
+    // Elements to manipulate iTowns and 3DTiles
+    this.itownsView = itownsView;
+    this.controls = controls;
+    this.layer = itownsView.getLayerById('3d-tiles-layer');
+    this.tbi = null;
```
Comment (jailln) [inline UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js]: Add a link to the doc please :)

File: UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+  }
+
+  /**
+   * If the user is currently hovering a building, fetches the building ID and
+   * displays it in the window.
+   * 
+   * @param {MouseEvent} event The mouse event.
+   */
+  onMouseMove(event) {
+    if (event.target.nodeName.toUpperCase() === 'CANVAS') {
```
Comment (jailln) [inline UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js]: A comment explaining this if would ease understanding

**Comment by Crejak** [comment]
Added commits to support link deletion + changed the "travel" functionnality to use centroid data from the link instead of the building information. Related to https://github.com/MEPP-team/UDV-server/pull/80 (maybe merge the server first).

File: UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js (hunk 2 above)
Comment (Crejak) [reply UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js]: Done

File: UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js (hunk 1 above)
Comment (Crejak) [reply UDV-Core/src/Extensions/DocumentLinks/views/DocumentLinkWindow.js]: Done

**Pull request merged**